packages/integration-karma/test/host-element/index.spec.js

```diff
@@ -0,0 +1,13 @@
+import { createElement } from 'lwc';
+import Shadow from 'x/shadow';
+
+describe('host element', () => {
+    it('should accept children being appended to it', () => {
+        const element = createElement('x-container', { is: Shadow });
+        document.body.appendChild(element);
+        if (process.env.NATIVE_SHADOW) {
+            // synthetic shadow returns null here even though it successfully inserts the div
```

Comment:
There are two reasons for this:
1. Root elements(the first LWC element in the tree) are [not expected to have slotted elements](https://github.com/salesforce/lwc/blob/master/packages/@lwc/synthetic-shadow/src/faux-shadow/element.ts#L360). synthetic-shadow has patched some APIs with this undocumented constraint
2. The manually added div is not being watched by the synthetic-shadow polyfill.